Add UWP as platform that supports .NET Standard

philippgille opened the pull request:
https://blogs.msdn.microsoft.com/dotnet/2017/10/10/announcing-uwp-support-for-net-standard-2-0/

philippgille committed:
Add UWP as platform that supports .NET Standard

https://blogs.msdn.microsoft.com/dotnet/2017/10/10/announcing-uwp-support-for-net-standard-2-0/

philippgille commented:
Well, the "Fall Creators Update" constraint isn't even necessary. UWP 10.0 supports .NET Standard 1.0, as seen in [this table](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support). What do you think?

jzeferino reviewed (commented):
LGTM. 
Also I recommended to add more platforms see #3814.

mairaw commented on docs/standard/class-libraries.md:
```diff
@@ -49,6 +49,7 @@ The .NET Standard does not expose the entire functionality of the .NET Framework
 The following platforms support .NET Standard libraries:
 
 *   .NET Core
+*   Universal Windows Platform (UWP), Fall Creators Update or newer
```
should we simply list the version here instead?
Universal Windows Platform (UWP) 10.0 and later versions

@terrajobst?

mairaw reviewed (changes requested):
Thanks @philippgille. Please see my comment regarding how I think we should list UWP here.

philippgille committed:
Add platforms; remove versions; add link

Instead of listing the same platforms that are listed on https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support and having to maintain both lists as new versions are released, just mention the platforms here and link to the existing table with versions.

Closes #3814

philippgille commented:
@mairaw: Thanks for the advice. I also thought about @jzeferino's advice. I decided that instead of listing all the platforms with their versions, where the info which .NET Standard version is supported is still missing, a listing with just the platform names *plus* a link to the [.NET Standard versions / .NET platforms versions matrix](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support) might be best. What do you think?

Also: Should I squash the commits?

philippgille commented:
@mairaw: Please see [my previous comment](https://github.com/dotnet/docs/pull/3813#issuecomment-348692601)

mairaw reviewed (changes requested):
Apologies on not getting back to you sooner @philippgille . I lost track of this PR. I left a small comment for you but changes overall look good. Let me know if you'd like me to address the feedback for you.

mairaw committed:
feedback
